tests/unit/amo/components/TestRatingManager.js

```diff
@@ -333,8 +333,8 @@ describe('RatingManager', () => {
     });
 
     it('cannot render a login prompt for unknown extension types', () => {
```

Comment:
can or cannot?

Reply:
🤦‍♂️

Reply:
yes "can" ... i'll update

Reply:
Cool, I think we should link to the original issue too.